### PR TITLE
Call for Mentors

### LINKED ISSUE #40: Call for Mentors
Write up a blog post that walks through what it’s like to be a mentor at HackIllinois and puts out an open call for mentors to join us at HackIllinois 2019.

### PR DESCRIPTION
**Pull request by justinyangusa**
Closes #40.

This post is about an open call for mentors to attend HackIllinois 2019. Potential mentors, many of whom are unfamiliar with HackIllinois, can reference this post to learn more about mentorship and HackIllinois itself. Interested contributors are invited to fill out the interest form to be contacted by Outreach. The release of this post is intended to be coordinated with social media announcements and posts on forums such as Hacker News and Reddit.

## Pre-Approval
- [x] Post properly titled
- [x] Post contains proper front matter
- [x] `featured` tag has been added to this post
- [ ] `featured` tag has been removed from other posts
- [ ] Images and assets have been added
- [ ] Post has been tested locally with Jekyll
- [ ] Spelling has been checked

## Pre-Deploy
- [ ] Reviews have been resolved or otherwise addressed
- [ ] Approval from a repo owner
- [ ] Post file name has been updated to current date
- [ ] Post front matter has been updated to current date
- [ ] Merged master

### CODE REVIEW

**Comment by apache8080** [comment]
@justinyangusa This looks good to me.

File: _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Mentors lead a group of students in addressing issues on projects that the mentors contribute to. Unlike at other collegiate hackathons, mentors don't just support students in fixing bugs here and there; mentors work together with students, guiding them through the open source workflow to make meaningful contributions.
+
+What sort of contributions have groups made in the past?
+In the past, HackIllinois has seen students make contributions to a projects in the areas of JavaScript, Android, Python, Linux, and more! Groups have opened PRs to projects such as PrairieLearn, a problem-driven learning system built at the University of Illinois; Puppeteer, Chrome's headless node API; and Rust, a systems programming language that's a favorite amongst developers. As a mentor, you can bring projects that you maintain or contribute to---anything you believe you can guide students through making contributions to.
+
+How many people will I be working with?
+Group sizes vary, but we find that the most successful groups have a ratio of 4-6 students per mentor and have 1-3 mentors that come ready with a handful features and fixes for the students to get started on.
```
Comment (redsn0w422) [inline _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown]: I would mention that the mentor gets to provide preference for their group size - we want them to be comfortable.

File: _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+What are the skill levels of the students?
+Although there is a wide range of skill levels, most students are well-versed with writing code for classes or personal projects, but don't necessarily have familiarity with working in larger codebases or open source workflows. Mentors get to help students become familiar with the open source aspects of development!
```
Comment (redsn0w422) [inline _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown]: I'd say:

"or open source workflows (like git)."

File: _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+What do I get out of attending HackIllinois?
+Mentors get to inspire and teach students about open source, helping them to become active members in the community. As a mentor, you would work closely with students to help resolve issues on your projects. Mentors also get to meet leaders and other mentors at the event. Of course, we will also provide food and snacks throughout the weekend, as well as tons of swag!
+Team members will be able provide details regarding travel and accommodations based on your individual situation following initial conversations.
```
Comment (redsn0w422) [inline _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown]: Which team?

File: _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+What happens when I fill out this Mentor Interest Form?
+We will review submissions, and a team member will reach out to you via email to learn more about your projects and answer any questions you may have. The Mentor Interest Form is a way for you to get connected with our team and learn more; there is no commitment to attend the event based on form submission.
```
Comment (redsn0w422) [inline _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown]: Again, I'd say "staff" or something.

**Review by redsn0w422 — CHANGES_REQUESTED** [review]
Good start. Couple inline comments, as well as more general things.

- I would define what we consider an open source contributor to be, and provide some high-level examples. Mentioning we've had people younger than 20 and college students would help.

- I think we should link to the mentor spotlights somehow, either by just linking if they want to read about someone's experiences, or even pulling some quotes out and scattering it throughout the post.

File: _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+How many people will I be working with?
+Group sizes vary, but we find that the most successful groups have a ratio of 4-6 students per mentor and have 1-3 mentors that come ready with a handful features and fixes for the students to get started on.
+
+What are the skill levels of the students?
+Although there is a wide range of skill levels, most students are well-versed with writing code for classes or personal projects, but don't necessarily have familiarity with working in larger codebases or open source workflows. Mentors get to help students become familiar with the open source aspects of development!
+
+What do I get out of attending HackIllinois?
```
Comment (apa-123) [inline _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown]: can add this line here - HackIllinois may subsidize travel/lodging costs on a case-by-case basis.

File: _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+Mentors get to inspire and teach students about open source, helping them to become active members in the community. As a mentor, you would work closely with students to help resolve issues on your projects. Mentors also get to meet leaders and other mentors at the event. Of course, we will also provide food and snacks throughout the weekend, as well as tons of swag!
+Team members will be able provide details regarding travel and accommodations based on your individual situation following initial conversations.
+
+What happens when I fill out this Mentor Interest Form?
+We will review submissions, and a team member will reach out to you via email to learn more about your projects and answer any questions you may have. The Mentor Interest Form is a way for you to get connected with our team and learn more; there is no commitment to attend the event based on form submission.
+
+I won't be able to make the dates this year, but I'd like to get updates about other events.
+Feel free to fill out the Mentor Interest Form anyway. We can contact you about other opportunities, such as our fall HackNights or HackIllinois in future years.
```
Comment (apa-123) [inline _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown]: potentially provide one more line here about HackNights to give more information

File: _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+I won't be able to attend, but I think this event would be great for others from my project or company.
+Feel free to share this blog post with others who may be interested in mentoring or shoot us an email at [opensource@hackillinois.org](mailto:opensource@hackillinois.org) to tell us about projects or companies that would be good fits!
+If you think your company would be interested in getting involved, please have your recruiting or technical directors reach out to us at [sponsor@hackillinois.org](mailto:sponsor@hackillinois.org).
+
+What if I have a question that wasn't answered here?
+We love questions! Mentors are the core of the HackIllinois experience and we work closely with each mentor. Just shoot us an email at [opensource@hackillinois.org](mailto:opensource@hackillinois.org) and someone on our team will get back to you within 24 hours.
```
Comment (apa-123) [inline _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown]: echoing the "team" comment from earlier, "staff" here as well

File: _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+Who? Open source contributors to projects big and small
+What? A 36-hour open source hackathon that brings together 1000+ students and mentors. Work throughout the event with a small group of students on open source contributions.
+When? Friday-Sunday, February 22-24, 2019
+Where? The University of Illinois at Urbana-Champaign, Urbana, Illinois, United States
+Approximately 150 mi / 250 km from each of Chicago, Illinois; St. Louis, Missouri; and Indianapolis, Indiana
+Why? HackIllinois aims to foster education, collaboration, and mentorship within the open source community. Come help students during their first forays into open source, and meet leaders and other contributors in open source.
+How? Just fill out the Mentor Interest Form: [go.hackillinois.org/mentor](http://go.hackillinois.org/mentor)
+
+What are the commitments and responsibilities of a mentor?
+Mentors lead a group of students in addressing issues on projects that the mentors contribute to. Unlike at other collegiate hackathons, mentors don't just support students in fixing bugs here and there; mentors work together with students, guiding them through the open source workflow to make meaningful contributions.
```
Comment (rohinb2) [inline _posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown]: addressing issues on projects **and** building new open source tools

**Review by rohinb2 — COMMENTED** [review]
Looks good! Pretty straightforward, seems fine to me.

**Review by smg155 — APPROVED** [review]
Looks pretty good to me!

**Pull request merged**